examples/webgl_loader_gainmap.html

```diff
@@ -0,0 +1,322 @@
+				let material = new THREE.MeshStandardMaterial( {
+					color: 0xffffff,
+					metalness: params.metalness,
+					roughness: params.roughness
+				} );
+
+				torusMesh = new THREE.Mesh( geometry, material );
+				scene.add( torusMesh );
+
+
+				geometry = new THREE.PlaneGeometry( 200, 200 );
+				material = new THREE.MeshBasicMaterial();
+
+				planeMesh = new THREE.Mesh( geometry, material );
+				planeMesh.position.y = - 50;
+				planeMesh.rotation.x = - Math.PI * 0.5;
+				scene.add( planeMesh );
+
+
+				const pmremGenerator = new THREE.PMREMGenerator( renderer );
+				pmremGenerator.compileEquirectangularShader();
+
+				THREE.DefaultLoadingManager.onLoad = function ( ) {
+
+					pmremGenerator.dispose();
+
+				};
+
+
+
+				gainMap = new HDRJPGLoader( renderer )
+					.load( 'textures/gainmap/spruit_sunrise_4k.jpg', function ( response ) {
+
+						resolutions[ 'HDR JPG' ] = response.width + 'x' + response.height;
+						displayStats( 'HDR JPG' );
+			
+						gainmapRenderTarget = pmremGenerator.fromEquirectangular( response.renderTarget.texture );
+			
+						gainmapBackground = response.toDataTexture();
+						gainmapBackground.mapping = THREE.EquirectangularReflectionMapping;
+						gainmapBackground.minFilter = THREE.LinearFilter;
+						gainmapBackground.magFilter = THREE.LinearFilter;
+						gainmapBackground.generateMipmaps = false;
+
+						gainmapBackground.needsUpdate = true;
+
+					}, function ( progress ) {
+
+						fileSizes[ 'HDR JPG' ] = humanFileSize( progress.total );
+			
+					} );
+
+				separateGainMap = new GainMapLoader( renderer )
+					.load( [ 'textures/gainmap/spruit_sunrise_4k.webp', 'textures/gainmap/spruit_sunrise_4k-gainmap.webp', 'textures/gainmap/spruit_sunrise_4k.json' ], function ( response ) {
+
+						resolutions[ 'Webp Gain map (separate)' ] = response.width + 'x' + response.height;
+			
+						separateGainmapRenderTarget = pmremGenerator.fromEquirectangular( response.renderTarget.texture );
+
+						separateGainmapBackground = response.toDataTexture();
+						separateGainmapBackground.mapping = THREE.EquirectangularReflectionMapping;
+						separateGainmapBackground.minFilter = THREE.LinearFilter;
+						separateGainmapBackground.magFilter = THREE.LinearFilter;
+						separateGainmapBackground.generateMipmaps = false;
+
+						separateGainmapBackground.needsUpdate = true;
+			
+					}, function ( progress ) {
+
+						fileSizes[ 'Webp Gain map (separate)' ] = humanFileSize( progress.total );
+
+					} );
+
+				hdrCubeMap = new RGBELoader()
+					.load( 'textures/gainmap/spruit_sunrise_1k.hdr', function ( response ) {
+
+						resolutions[ 'HDR' ] = response.image.width + 'x' + response.image.height;
+			
+						hdrCubeRenderTarget = pmremGenerator.fromEquirectangular( response );
+
+						response.mapping = THREE.EquirectangularReflectionMapping;
+						response.minFilter = THREE.LinearFilter;
+						response.magFilter = THREE.LinearFilter;
+						response.needsUpdate = true;
```

Comment:
I'm not sure it make sense to configure `response` at that point. It seems the equirectangular texture is only used to produce the PMREM.

Reply:
it's more the  what happens [in webgl_materials_envmaps_hdr.html](https://github.com/mrdoob/three.js/blob/4567ecee2aa19584533260d331fd80d8100b2240/examples/webgl_materials_envmaps_hdr.html#L104C12-L104C12) only with the names changed from `hdrCubeMap` (parent scope name) to `response` (callback scope) and an added `minFilter`.

The response is used later when switching to "HDR" in the GUI as scene background (like the other example)

If it is not clear I can switch back to having the global parent scope's name

Reply:
But you create the `hdrCubeRenderTarget` variable from `response`. `hdrCubeRenderTarget` is later used in the code. Hence, there is no need to configure `response`.

Reply:
There, I made some renaming for consistency (cubeMap->equirectangularMap) and clarity (no more `response` in the loaded callbacks) 
![image](https://github.com/mrdoob/three.js/assets/19731017/3f4e61e5-852d-41bd-88dc-80cf6108f4cf)
![image](https://github.com/mrdoob/three.js/assets/19731017/5a0a5bdc-11bc-4646-b55b-60973a975cad)

I hope this is more clear, `hdrEquirectangularMap` is used later in the render Loop together with `hdrPMREMRenderTarget`, it needs configuring because it is the `scene.background`